setup.py

```diff
@@ -42,7 +42,8 @@
         'django_readonly_field',
     ],
     include_package_data=True,
-    install_requires=[],
+    install_requires=["Django>=1.8,<1.11"],
```

Comment:
It's always a debate in the Python community whether or not you should "future proof" your requirements.
My stand is that you shouldn't pin your requirements unless you know there is an issue with that version. That way you don't go in the way of the people installing your lib.